netbeans.apache.org/src/content/templates/footer.gsp

```diff
@@ -40,18 +43,18 @@
                 <% /* See https://www.apache.org/foundation/marks/pmcs#navigation */ %>
                 <h1>About</h1>
                 <ul>
-                    <li><a href="http://www.apache.org/foundation/thanks.html">Thanks</a></li>
-                    <li><a href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
-                    <li><a href="http://www.apache.org/security/">Security</a></li>
-                    <li><a href="http://incubator.apache.org/projects/netbeans.html">Incubation Status</a></li>
+                    <li><a href="https://www.apache.org/foundation/thanks.html">Thanks</a></li>
+                    <li><a href="https://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
+                    <li><a href="https://www.apache.org/security/">Security</a></li>
+                    <li><a href="https://incubator.apache.org/projects/netbeans.html">Incubation Status</a></li>
                 </ul>
             </div>
             <div class="large-auto cell">
                 <h1><a href="/community/index.html">Community</a></h1>
                 <ul>
                     <li><a href="/community/mailing-lists.html">Mailing lists</a></li>
-                    <li><a href="/community/committer.html">Becoming a committer</a></li>
-                    <li><a href="/community/events.html">NetBeans Events</a></li>
+                    <li><a href="/community/committer.html">Becoming a commiter</a></li>
```

Comment:
Typo

Reply:
I'll be writing "committer" with two 'm's and two 't's one hundred times in my blackboard. Thanks.